infra/Pos/DHT/Real/Real.hs

```diff
@@ -145,12 +147,15 @@ kademliaGetKnownPeers
        , Bi DHTKey
        )
     => KademliaDHTInstance
-    -> m [NetworkAddress]
+    -> m [Peer]
 kademliaGetKnownPeers inst = do
     let kInst = kdiHandle inst
     let initNetAddrs = bool [] (kdiInitialPeers inst) (kdiExplicitInitial inst)
     buckets <- liftIO (K.viewBuckets $ kInst)
-    extendPeers (kdiKey inst) initNetAddrs buckets
+    addresses <- extendPeers (kdiKey inst) initNetAddrs buckets
+    -- TBD kademlia node classification? We'll assume they're all relay nodes.
```

Comment:
I think initially for integration of this stuff we should have all nodes believing they and all peers are core nodes. Otherwise we're going to have differences in policy while we're still converting all uses of getPeers / converseWithNeighbours.